packages/tools/kda-cli/readme.md

```diff
@@ -0,0 +1,77 @@
+# kda-cli
+
+This cli tool is here to assist you with your development on the kadena blockchain.
+
+## Prerequisite
+
+In order to use this cli tool the following packages need to be installed manually.
+In a later version some might be installed via the cli tool.
+
+- [node v16+](https://nodejs.org/en)
+- [docker](https://docs.docker.com/get-docker/)
+
+Login to docker with your
+[github account](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)
+, use the gh access token as password:
+
+```
+docker login ghcr.io -u <your gh username>
+```
+
+## Install
+
+Note that the package is not yet published, so for now you need to clone this
+repo and make a symlink.
+
+```bash
+$ npm install --global kda-cli
+```
+
+### Install From repo
+
+To install the executable from this repo:
+
+```bash
+rush install && rushx build
+chmod +x ./lib/cli.js
+# if you are using NVM, you should have this environment variable available
+ln -s $(pwd)/lib/cli.js $NVM_BIN/kda-cli
+# if not, you can replace $NVM_BIN to any path you have added in your $PATH
+```
+
+## CLI
+
+```
+$ kda-cli --help
```

Comment:
`kda` is different compared to `kda --help`, I needed the former to find `list` and `init`, which work :)

```
❯ kda list
{ 'vue-h5-template': 'wawsc5354524/vue-h5-template' }
```

```
❯ kda init vue-h5-template kda-test-project

 Start generating... 

✔ Downloading...

 Congratulation! Generation completed!

 To get started

 cd kda-test-project,and npm i or yarn to install dependencies!
```

Last line of message could be improved (separate lines).

Also:

```
❯ npm run 
docs         docs:deploy  
```

```
❯ npm run docs

> vue-cli-template-webpack@1.3.1 docs
> cd docs && gitbook serve

sh: gitbook: command not found
```